pinc/3rdparty/mediawiki/README.md

```diff
@@ -35,19 +35,20 @@ To use them, see ../DifferenceEngineWrapper.inc
 
 To update the files from MediaWiki:
 1. Check out MediaWiki code from git
-  `git clone https://gerrit.wikimedia.org/r/p/mediawiki/core.git mediawiki`
+  `git clone --depth 1 https://gerrit.wikimedia.org/r/mediawiki/core mediawiki`
+  `git fetch --tags`
 2. Show list of release tags
-  `git tags`
+  `git tag -l`
 3. Change to the release you want to use
   `git checkout <tag_name>`
 4. Copy updated files from locations mentioned above to this directory
-5. Unless the code has been fully converted to UTF-8:
+6. Unless the code has been fully converted to UTF-8:
```

Comment:
should it be 5 again?

Reply:
Woops. Good point. I've updated all the other numbers.